neqo-transport/src/crypto.rs

```diff
@@ -861,115 +861,132 @@ impl CryptoStreams {
     }
 
     pub fn send(&mut self, space: PNSpace, data: &[u8]) {
-        self[space].tx.send(data);
+        if let Some(cs) = self.get_mut(space) {
+            cs.tx.send(data);
+        } else {
+            panic!("Trying to send in a dropped space")
+        }
     }
 
     pub fn inbound_frame(&mut self, space: PNSpace, offset: u64, data: Vec<u8>) -> Res<()> {
-        self[space].rx.inbound_frame(offset, data)
+        if let Some(cs) = self.get_mut(space) {
+            cs.rx.inbound_frame(offset, data)
+        } else {
+            // TODO should be panic()? How did we read a packet from a dropped space?
+            qinfo!("Inbound frame in a dropped space");
+            Ok(())
```

Comment:
This is an inbound frame, but if we have discarded the cryptographic state, then we shouldn't be able to decrypt the packet.  I think we can panic.  But see above regarding unwrap().